layouts/BeltDocsFlavour.re

```diff
@@ -0,0 +1,74 @@
+module Flavour = {
+  type t =
+    | Reason
+    | OCaml
+    | Other;
+
+  let toReadableString =
+    fun
+    | Reason => "RE"
+    | OCaml => "ML"
+    | Other => "Other";
+
+  let fromString =
+    fun
+    | "re"
+    | "reason" => Reason
+    | "ml"
+    | "ocaml" => OCaml
+    | _ => Other;
+};
+
+type t = {
+  flavour: Flavour.t,
+  setFlavour: Flavour.t => unit,
+  refmt: (~lang: string, string) => string,
+};
+
+let defaultValue = {
+  flavour: Reason,
+  setFlavour: _ => (),
+  refmt: (~lang as _, src) => src,
```

Comment:
We need a way to also differentiate between `interface` (rei / mli) and `implementation` (re/ml). We have codeblocks with a `sig` metastring which should be interpreted as mli / rei code, so we have a way to pass a flag

Reply:
Good point!